person.rb

```diff
@@ -1,14 +1,23 @@
-class Person
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
* I get an error here when I try to test your decoration, I think the issue is that you are setting the name argument to unknown so I noticed when I changed it to ` name = "unknown" it worked. check the screenshots below ⬇️ 
* Also I noticed your trimmer method isn't functioning as expected as you can see in the screenshot the word was not trimmed , Kindly work on these 🙏 
![image](https://user-images.githubusercontent.com/52497246/210695089-aca62c41-10fa-46dd-bd5b-1bb2f007ba0d.png)
![image](https://user-images.githubusercontent.com/52497246/210695444-5b461cf8-d887-45a8-9c14-d6ac8101edcd.png)